setup.py

```diff
@@ -18,6 +19,13 @@ def local_file(name: str) -> Path:
         raise AssertionError("__version__ must be set")
 
 
+if "FLAKE8_TRIO_NO_REGISTER" in os.environ and os.environ["FLAKE8_TRIO_NO_REGISTER"]:
```

Comment:
```suggestion
if os.getenv("FLAKE8_TRIO_NO_REGISTER"):
```

Defaults to None if not present